src/ZeekConfig.cmake.in

```diff
@@ -0,0 +1,33 @@
+@PACKAGE_INIT@
+
+include(CMakeFindDependencyMacro)
+
+set(CMAKE_THREAD_PREFER_PTHREAD ON)
+set(THREADS_PREFER_PTHREAD_FLAG ON)
+find_dependency(Threads REQUIRED)
+
+find_dependency(OpenSSL REQUIRED)
+
+if ( MSVC )
```

Comment:
Add a comment here that for MSVC, these dependencies come from Conan which is why they're spelled out explicitly.